CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-- Your favourite command isn't covered?
+- Your favorite command isn't covered?
```

Comment:
Meh :smiley: `tldr` started in British english, but I'm not too fussed if you feel like changing it.